package.json

```diff
@@ -4,22 +4,22 @@
   "version": "0.0.1",
   "license": "MIT",
   "scripts": {
-    "build:demos": "node ./scripts/run.js prepublish demos",
-    "checkdeps": "node ./scripts/dependencies.test.js",
+    "precommit": "lint-staged",
     "commit": "git-cz",
     "commitmsg": "node ./node_modules/cz-customizable-ghooks/lib/index.js .git/COMMIT_EDITMSG",
     "coverage": "node ./scripts/run.js coverage",
     "coverage:upload": "cat ./coverage/lcov.info | coveralls",
-    "deploy": "cd website/publisher && npm run ci",
-    "link": "node ./scripts/run.js link",
-    "lint": "standard",
     "postcoverage": "istanbul-combine -d coverage -p summary -r lcov -r html packages/**/coverage/*.json babel/**/coverage/*.json",
     "postinstall": "npm run setup",
-    "postsetup": "npm run link",
-    "prelint": "npm run checkdeps",
-    "pretest": "npm run lint",
     "setup": "node ./scripts/run.js prepublish nodemos",
-    "test": "node ./scripts/run.js test"
+    "build:demos": "node ./scripts/run.js prepublish demos",
+    "link": "node ./scripts/run.js link",
+    "postsetup": "npm run link",
+    "checkdeps": "node ./scripts/dependencies.test.js",
+    "pretest": "npm run checkdeps",
+    "test": "node ./scripts/run.js test",
+    "deploy": "cd website/publisher && npm run ci",
+    "format": "prettier-eslint --write \"**/*.js\"&&prettier-eslint --write \"packages/node_modules/**/*.js\""
```

Comment:
@henri-hulski it's very hard to reason logic in this case. I have tried it and there always was ambiguous result. So seems alphabetic order is only option to have stable unopinionated order.